tests/unit/amo/reducers/test_home.js

```diff
@@ -104,7 +104,9 @@ describe(__filename, () => {
     it('loads hero shelves', () => {
       const { store } = dispatchClientMetadata();
 
-      const heroShelves = createHeroShelves();
+      const heroShelves = createHeroShelves({
+        primaryProps: { addon: fakeAddon },
```

Comment:
Since `addon` is not relevant to this test, it would benefit from a `_createHeroShelves()` helper like you added to the other suite. Removing irrelevant setup data makes tests more readable.